wordpress-mobile/WordPress-iOS
# No search results: Wrap message

Fixes #10792 

To test:
- On a small device, go to a search that displays a no results message (ex: Media, Site Pages, Blog Posts).
- Enter an invalid search term.
- Verify the message is not truncated.

<img width="300" alt="no_results_message" src="https://user-images.githubusercontent.com/1816888/50993900-fc9a3700-14d7-11e9-8aa3-2408f8d85b14.png">


Update release notes:
